### PR TITLE
Add queries for new examples

### PR DESCRIPTION
**Pull request by arseniy-sonar**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -26,11 +26,18 @@ Running on the following CI services:
...
 
 Configured for analysis on:
 
 * https://github.com/sonarsource-cfamily-examples?q=-sq[SonarQube]
 * https://github.com/sonarsource-cfamily-examples?q=-sc[SonarCloud]
 
+You can find also a few examples demonstrating
+
+* https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=compdb[the use of Compilation Database (compile_commands.json)]
+* https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=topic%3Acoverage[test coverage gathering]
```
Comment (amelie-renard-sonarsource) [inline README.adoc]:
```suggestion
* https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=compdb[the use of Compilation Database (compile_commands.json)]
* https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=topic%3Acoverage[Test coverage]
```

File: README.adoc [hunk 2]
```diff
@@ -26,11 +26,18 @@ Running on the following CI services:
 * https://github.com/sonarsource-cfamily-examples?q=jenkins[Jenkins]
 * https://github.com/sonarsource-cfamily-examples?q=gitlab[GitLab CI]
 * https://github.com/sonarsource-cfamily-examples?q=bitbucket[BitBucket Pipelines]
+* plus https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=otherci[generic CI]
+* and https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=manual[manually]
```
Comment (amelie-renard-sonarsource) [inline README.adoc]:
```suggestion
Additionally, you can find [generic examples](https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=otherci[generic examples]) that work for other CIs as well as [manual configurations](https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=manual[manual configurations]) of SonarQube.
```

Comment (amelie-renard-sonarsource) [reply README.adoc]: I don't find the fact of adding these two items to the list very clear. I'd instead write them down in a sentence like the one above. WDYT?

**Review by amelie-renard-sonarsource — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 2 above)
Comment (arseniy-sonar) [reply README.adoc]: I think adding a top-level entry like this breaks the flow of the description. I tried to clarify it, PTAL.

File: README.adoc (hunk 1 above)
Comment (arseniy-sonar) [reply README.adoc]: Reformulated.

File: README.adoc (hunk 2 above)
Comment (amelie-renard-sonarsource) [reply README.adoc]: Perfect!

**Pull request merged**